Source/Laser/LaserProfilesImpl/LaserProfileFromTXYEFile.cpp

```diff
@@ -133,7 +133,7 @@ WarpXLaserProfiles::FromTXYEFileLaserProfile::parse_txye_file(std::string txye_f
 
         if(m_params.nt <= 1) Abort("nt in txye file must be >=2");
         if(m_params.nx <= 1) Abort("nx in txye file must be >=2");
-#if (AMREX_SPACEDIM == 3)
+#if ((AMREX_SPACEDIM == 3) || (defined WARPX_DIM_RZ))
```

Comment:
@hightower8083 @lucafedeli88 is it possible that the TXYEFile laser has no coverage for RZ in the tests? Do you like to add one with the PR?

Reply:
form the view of txye-file its just a 3D. It seems that tests components are already in the [python script](https://github.com/ECP-WarpX/WarpX/blob/development/Examples/Modules/laser_injection_from_file/analysis.py), so we just need to add the function
```python
def create_gaussian_3d():
   T, X, Y = np.meshgrid(tcoords, xcoords, xcoords, indexing='ij')
   E_t = gauss(T,X,Y,'3d')
   write_file("gauss_3d.txye", xcoords, xcoords), tcoords, E_t)
   write_file_unf("gauss_3d_unf.txye", xcoords, xcoords, tcoords, E_t)
```
assert usage of proper slice in `do_analysis` and add an input script. I can check this when i have time (not right now..)

Reply:
Yes, there is no test for a TXYE file in 3D. Actually, there is a PR  on that ( https://github.com/ECP-WarpX/WarpX/pull/669 ). It was opened one year ago, but it was never merged because we agreed to wait for an update of `regression_testing`. The PR itself could be completed with very little effort.